app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        if 1.0 <= distance_from_city_center <= 10.0:
+            self.distance_from_city_center = distance_from_city_center
+        if 1 <= clean_power <= 10:
+            self.clean_power = clean_power
+        if 1.0 <= average_rating <= 5.0:
+            self.average_rating = average_rating
+        if count_of_ratings > 0:
+            self.count_of_ratings = count_of_ratings
```

Comment:
What if some user runs your program via terminal(let's imagine we can do it) and enters 100, how you program will behave? As you set these attributes only when these conditions are met, it means in our case the class instance won't have them and user will get AttributeError when other methods will be called. So you should handle this case